client/me/help/main.jsx

```diff
@@ -127,29 +151,39 @@ const Help = React.createClass( {
 	render: function() {
 		const {
 			isEmailVerified,
-			showCoursesTeaser
+			userId
 		} = this.props;
 		return (
 			<Main className="help">
 				<MeSidebarNavigation />
 				<HelpSearch />
 				{ ! isEmailVerified && <HelpUnverifiedWarning /> }
-				{ showCoursesTeaser && this.getCoursesTeaser() }
+				{ this.getCoursesTeaser() }
 				{ this.getHelpfulArticles() }
 				{ this.getSupportLinks() }
 				<HappinessEngineers />
+				<QueryUserPurchases userId={ userId } />
 			</Main>
 		);
 	}
 } );
 
-module.exports = reactRedux.connect(
+export default connect(
 	( state, ownProps ) => {
-		const isEmailVerified = currentUser.isCurrentUserEmailVerified( state );
-		const showCoursesTeaser = ownProps.isCoursesEnabled;
+		const isEmailVerified = isCurrentUserEmailVerified( state );
+		const userId = getCurrentUserId( state );
+		const purchases = getUserPurchases( state, userId );
+		const isLoading = ! hasLoadedUserPurchasesFromServer( state );
```

Comment:
Perhaps `isFetchingUserPurchases` would be a better fit here?

It's unlikely, but if loading fails I have a feeling that the placeholder would keep flashing forever, while `isFetchingUserPurchases` should stop flashing if it gives up.